packages/types/src/organizationMembership.ts

```diff
@@ -40,7 +40,9 @@ export interface OrganizationMembershipResource extends ClerkResource {
   update: (updateParams: UpdateOrganizationMembershipParams) => Promise<OrganizationMembershipResource>;
 }
 
-export type MembershipRole = 'admin' | 'basic_member' | 'guest_member';
+// Adding (string & {}) allows for getting eslint autocomplete but also accepts any string
```

Comment:
☁️ We probably need to extract this (in a separate cleanup task) to a `AnyString` (or something else meaningful) type and using somewhere to avoid adding comments about what this syntax means all over the codebase.

Reply:
i'll add a ticket